### PR TITLE
updated Overview article

### PR DESCRIPTION
**Pull request by Vegebutcher**
previously only old screenshot, now updated with description

### CODE REVIEW

File: g1ant.robot-window/overview.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 
+1. The **workspace** a.k.a. the script editor is the place where you type your code. The workspace is tabbed: you can have many robot scripts opened at the same time, each in its own tab.
+
+2. [**Panels**](panels.md) provide additional information related to the G1ANT.Language and your script. On the screenshot above, on the left you can see the Addons panel along with the list of a selected addon elements, the Variables panel at the bottom, which displays all variables currently in use by a script, with their values and structure information, and the Windows Tree panel on the right displaying information on opened windows and their elements.
+
+   You can show/hide the panels by selecting them from the `View` menu.
+
+3. **Status bar** displays the following information, starting from the left:
+
+   - Program status: *“Ready”* means that the robot is able to execute process, *“Unable to execute process”* means that an error has occurred or the process is already running.
```
Comment (WiktoriaPrusik) [inline g1ant.robot-window/overview.md]: I don't recognise “Unable to execute process” message. Where does it occur? Are you sure it is an updated info? :) If process is already running or if an error occurred, the status is displayed differently on my machine

**Review by WiktoriaPrusik — APPROVED** [review]
please look at my comment

File: g1ant.robot-window/overview.md (hunk 1 above)
Comment (Vegebutcher) [reply g1ant.robot-window/overview.md]: This message was copied from the old manual, so probably it's outdated.

Comment (WiktoriaPrusik) [reply g1ant.robot-window/overview.md]: Okay so can you correct it, please? For example: "Process has finished" is a common status

**Review by lfronczyk — APPROVED** [review]

**Pull request merged**